src/java/org/apache/cassandra/cql3/TokenRelation.java

```diff
@@ -124,6 +124,12 @@ protected Restriction newAnnRestriction(TableMetadata table, VariableSpecificati
         throw invalidRequest("%s cannot be used for toekn relations", operator());
     }
 
+    @Override
+    protected Restriction newAnalyzerMatchesRestriction(TableMetadata table, VariableSpecifications boundNames)
+    {
+        throw invalidRequest("%s cannot be used for token relations", operator());
```

Comment:
I think you meant tokenized?

But let's stick with the terms `analyzer/analyzed` since it may not actually involve tokenization